src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -44,7 +44,8 @@ if [ \"\$(ulimit -c)\" != \"unlimited\" ]; then
 fi
 echo \"/var/tmp/core.%e.%t.%p\" > /proc/sys/kernel/core_pattern"
 rm -f /var/tmp/core.*
-if [ "${HOSTNAME%%.*}" != "$FIRST_NODE" ]; then
+if [ "${HOSTNAME%%.*}" != "$FIRST_NODE" ] ||
```

Comment:
I've been wondering if we really need the handling of /mnt/daos in this script.  There is no guarantee that a test will actually use this specific mount point (the HW Medium tests typically use `/mnt/daos0` and `/mtn/daos1`).  Last time I ran tests manually the normal server startup handled the mount point creation w/o it needing to be created beforehand.

Reply:
Well, as you can see, this PR is removing some of the handling of `/mnt/daos` in this script, at least for the case where we don't want it to be a _tmpfs_ because we have pmem.

I would not object to a PR that removes it entirely although (not that I have given it much thought but) it's not clear in my mind where the _tmpfs_ option for `/mnt/daos` would happen for VM clusters.